### PR TITLE
Add Nov 2017 Planning Session

### PR DESCRIPTION
**Pull request by dasanchez**
Planning meeting set for November 23, 2017

### CODE REVIEW

**Review by dcwalk — COMMENTED** [review]
I need to confirm that we can use the space on this night, unless you have another place in mind that we could host

**Comment by dasanchez** [comment]
dcwalk: Sorry, I should have asked first! Can you confirm please? If it's a no-go for that evening but there's an alternative I'll change the date. If it's a no-go altogether I'll start looking into libraries.

**Comment by dcwalk** [comment]
Okay I got the go ahead, we are good 👍

**Comment by dasanchez** [comment]
Thanks for the assist!

File: _posts/2017-11-23-planning-session.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+text: Winter planning session for Toronto Mesh.
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-11-23
+startTime: '18:30'
+endTime: '21:30'
+---
+
+Join us at our fifth planning session, where we regroup and plan for the winter! At this session we set priorities and projects and form **working groups** for the next six months and identify those interested in leading projects.  
+You can see our [current working groups and governance processes](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md) to get a sense of existing interests!
```
Comment (garrying) [inline _posts/2017-11-23-planning-session.md]: Should this be a paragraph? The line break seems out of place here

**Review by dcwalk — APPROVED** [review]

**Comment by dcwalk** [comment]
Looks like the travis errors are for hacklab.to I'd say we are good to go, will merge in a few but wait for others first

**Pull request merged**